### PR TITLE
Adds "setSelectionRange" to TextField component

### PR DESCRIPTION
**Pull request by martellaj**
#### Description of changes

- Adds `setSelectionRange` to `TextField` component.
  - Fix for `TextField` on iOS.
- Includes test.

### CODE REVIEW

**Comment by martellaj** [comment]
@dzearing I haven't generated the change file yet. Is this a minor or patch change?

**Comment by micahgodbolt** [comment]
@martellaj you've introduced a new publicly accessible function, so definitely minor.

**Comment by micahgodbolt** [comment]
Also, can you add this function to ITextField in the props file as well?

File: common/changes/@uifabric/example-app-base/joem-textfield-add-setSelectRange_2017-05-18-21-16.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (cschleiden) [inline common/changes/@uifabric/example-app-base/joem-textfield-add-setSelectRange_2017-05-18-21-16.json]: hm, you shouldn't need all these `type: none` change files

Comment (martellaj) [inline common/changes/@uifabric/example-app-base/joem-textfield-add-setSelectRange_2017-05-18-21-16.json]: It was the auto-output from rush. Is it cool to merge? I assume they just won't bump any of those packages anyway.

**Comment by micahgodbolt** [comment]
Yeah it's fine. It's a semi bug of rush as it compares your branch to origin by default. You can get a cleaner file by running rush change -b upstream

________________________________

**Comment by martellaj** [comment]
Thanks, @micahgodbolt. Good to know for future PRs.

**Comment by martellaj** [comment]
@micahgodbolt, please let me know if y'all need any other changes prior to merge. Thanks!

**Pull request merged**